pkg/ipc/ipc.go

```diff
@@ -627,6 +627,7 @@ func makeCommand(pid int, bin []string, config *Config, inFile, outFile *os.File
 			for {
 				n, err := rp.Read(output[size:])
 				if n > 0 {
+					os.Stdout.Write(output[size : size+uint64(n)])
```

Comment:
This all goes to the crash log files, which go to kernel developers and are intended for reproduction.
Currently we print only executed programs, and no internal/debug logs. I afraid of unconditionally polluting them with all kinds of debug info produced during normal operation. This will reduce number of executed programs captured and make the logs less readable.
This will also affect all tools and programs that execute programs.
What problem are you trying to solve? What important may be in that output?

Reply:
I am investigating these two cases:
https://syzkaller.appspot.com/bug?id=1d2ff914ef25aab39bbb4bfdb0cad79a37a0cbf5
https://syzkaller.appspot.com/bug?id=0834b576293cf07e25bc8b4188979f6a61e1bc38

From traces, I found that one executor processes was zombie. With the wait goroutine, I saw that it exited with 67. Then I decided to print executor logs and the case stopped reproducing, but I started to see more SYZFAIL reports:
 $ cat  /mnt/syzkaller/workdir/crashes/*/desc*
SYZFAIL: event already set
SYZFAIL: bad thread state in completion

Reply:
> From traces, I found that one executor processes was zombie. With the wait goroutine, I saw that it exited with 67.

I don't follow. Do you mean that starting the cmd.Wait goroutine earlier (in makeCommand) fixes the zombie processes?
If yes, then we should move it in a separate patch (w/o logging).

Reply:
> Then I decided to print executor logs and the case stopped reproducing, but I started to see more SYZFAIL reports:

Do you mean that adding printing causes more of SYZFAIL errors?

A low rate of SYZFAIL bugs is OK. The executor process can easily corrupt own address space, and if it will corrupt itself in a very special way, it will cause SYZFAIL.
Do you see low or high rate of these during fuzzing? There is #2787 to unify all of them to not produce lots of separate bug reports.